packages/foundry-sdk-generator/src/generate/betaClient/minifyBundle.ts

```diff
@@ -395,7 +565,24 @@ export class ProjectMinifier {
             continue;
           }
 
-          for (const namedExport of exportDeclaration.getNamedExports()) {
+          perNamedExport: for (
+            const namedExport of exportDeclaration.getNamedExports()
+          ) {
+            for (const n of nodesToKeepForTargetModule) {
+              if (
+                Node.isExportable(n)
+                && n.getSourceFile().getFilePath().startsWith("/internal")
+              ) {
+                if (n.getSymbol()?.getName() === namedExport.getName()) {
+                  continue perNamedExport;
```

Comment:
The existing code keeps a distinction between "nodes that we need to keep" based on whether it is in "this file" or in "that file". So we need to make sure that we do not end up `remove()`ing things if its in either list.